server/views/pages/scan/scan-barcode-result.njk

```diff
@@ -0,0 +1,36 @@
+{% extends "layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% set pageTitle = applicationName + " - Result of scanning a barcode" %}
+{% set pageId = 'scan-barcode-result' %}
+
+{% block content %}
+
+  <section class="app-container govuk-body">
+    {% if errors.length > 0 %}
+      {{ govukErrorSummary({
+        titleText: 'There is a problem',
+        errorList: errors,
+        attributes: { 'data-qa-errors': true }
+      }) }}
+    {% endif %}
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
```

Comment:
The view is deliberately basic. What I wanted to achieve in this PR was the service call and rendering a different heading on this page based on the API response (happy path vs. DUPLICATE)

The next PR will pretty the view up, adding the rest of the content etc